hack/kiali/func-sm.sh

```
@@ -0,0 +1,369 @@
+    local readyReplicas="0"
+    while [ "$?" != "0" -o "$readyReplicas" == "0" ]
+    do
+      sleep 1
+      echo -n '.'
+      readyReplicas="$(${OC} get ${op} -n ${OLM_OPERATORS_NAMESPACE} -o jsonpath='{.status.readyReplicas}' 2> /dev/null)"
+    done
+    echo "done."
+  done
+
+  infomsg "Wait for the servicemesh operator to be Ready."
+  ${OC} wait --for condition=Ready $(${OC} get pod -n ${OLM_OPERATORS_NAMESPACE} -o name | grep -E 'sail|servicemesh|istio') --timeout 300s -n ${OLM_OPERATORS_NAMESPACE}
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)